pkg/runtime/core/trainingruntime.go

```diff
@@ -20,6 +20,7 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	jobsetv1alpha2 "sigs.k8s.io/jobset/api/jobset/v1alpha2"
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
```suggestion

	jobsetv1alpha2 "sigs.k8s.io/jobset/api/jobset/v1alpha2"
	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Grouping dependency would be better

```
- standard lib

- third-party lib

- trainer lib
```